paper.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'HOHQMesh: An all Quadrilateral/Hexahedral Unstructured Mesh Generator for High Order Elements'
+tags:
+  - Mesh Generation
+  - Spectral Element Methods
+  - High Order
+  - Unstructured Quadrilateral and Hexahedral Meshes
+authors:
+  - name: David A. Kopriva
+    orcid: 0000-0002-8076-0856
+    affiliation: "1, 2"
+  - name: Andrew R. Winters
+    orcid: 0000-0002-5902-1522
+    corresponding: true
+    affiliation: 3
+  - name: Michael Schlottke-Lakemper
+    orcid: 0000-0002-3195-2536
+    affiliation: 4
+  - name: Joseph A. Schoonover
+    orcid: 0000-0001-5650-7095
+    affiliation: 5
+affiliations:
+ - name: Professor Emeritus, The Florida State University, Tallahassee, FL, USA
+   index: 1
+ - name: Adjunct Professor, San Diego State University, San Diego, CA, USA
+   index: 2
+ - name: Senior Associate Professor, Department of Mathematics; Applied Mathematics, Linköping University, Sweden
+   index: 3
+ - name: Professor, High-Performance Scientific Computing & Centre for Advanced Analytics and Predictive Sciences, University of Augsburg, Germany
+   index: 4
+ - name: Research Software Engineer, Fluid Numerics, Hickory, NC, USA
```

Comment:
As far as I have seen, it is not necessary/common to add the positions of the people involved. For example, we could just write
```suggestion
 - name: San Diego State University, San Diego, CA, USA
   index: 2
 - name: Department of Mathematics; Applied Mathematics, Linköping University, Sweden
   index: 3
 - name: High-Performance Scientific Computing & Centre for Advanced Analytics and Predictive Sciences, University of Augsburg, Germany
   index: 4
 - name: Fluid Numerics, Hickory, NC, USA
```

What do you think?